apps/meteor/app/channel-settings/server/methods/saveRoomSettings.ts

```diff
@@ -1,10 +1,11 @@
 import { Meteor } from 'meteor/meteor';
 import { Match } from 'meteor/check';
+import type { IRoom, RoomType, IRoomWithRetentionPolicy, IUser } from '@rocket.chat/core-typings';
 import { TEAM_TYPE } from '@rocket.chat/core-typings';
+import { Rooms } from '@rocket.chat/models';
```

Comment:
If you change the code to use the raw model then you can't assume that everything will be running in sync anymore.